### PR TITLE
Tests and ci

### PR DESCRIPTION
**Pull request by bendichter**

### CODE REVIEW

File: tests/test_ibl_converter.py [hunk 1]
```diff
@@ -0,0 +1,154 @@
+from pynwb import NWBHDF5IO, TimeSeries
+from .utils import *
```
Comment (bendichter) [inline tests/test_ibl_converter.py:2]: I'd rather not use `import *` outside of `__init__.py` files.

**Comment by bendichter** [comment]
@Saksham20 this looks really good!

**Comment by bendichter** [comment]
still needs:
* setup.py
* CI from GitHub Actions

File: tests/test_ibl_converter.py (hunk 1 above)
Comment (Saksham20) [reply tests/test_ibl_converter.py]: solved

**Comment by Saksham20** [comment]
> still needs:
> 
> * setup.py
> * CI from GitHub Actions

I have pushed both files, I need help setting up the CI for a conda env, is that possible? Currently its just a generic file from another package: we need to update it.

**Comment by bendichter** [comment]
Why do you need conda? Why not just make it pip installable?

**Comment by Saksham20** [comment]
ibl recommends to make a conda env : https://github.com/int-brain-lab/iblenv
and I had had issue installing pyarrow with pip but worked with conda https://github.com/int-brain-lab/iblenv/issues/40
without conda it wouldnt run on my machine, but gitactions uses ubuntu right? so it might work

**Comment by bendichter** [comment]
These tests require a connection to the IBL database, which isn't really possible for CI. See https://alan-turing-institute.github.io/rsd-engineeringcourse/ch03tests/05Mocks.html

**Comment by Saksham20** [comment]
> These tests require a connection to the IBL database, which isn't really possible for CI given permission, and could fail based on service interruptions. Maybe try using mocks. See https://alan-turing-institute.github.io/rsd-engineeringcourse/ch03tests/05Mocks.html

I've added functionality to catch HTTP error or connection error (which come up if login credentials is wrong/404 err or if the ip address is not registered). Test will pass despite these errors. https://github.com/catalystneuro/IBL-to-nwb/pull/8/commits/10437cd90bfb5cdf9f7220f5b5848bde777e853c
I checked mock functions but dont see a definitive need here over the above exception handling. 
Also for CI, 
- I cant think of a way to use it since IBL servers require login credentials: how to hide them in the repo once passed in as args) 
- and the IP address of the requesting computer needs ot be registered: dont know how we could do that for the ubuntu running for CI testing. 

Other option is to store the json file and a nwb file in the repo for testing. This nwb file would be 500MB ish which can make the repo heavy. Is there a workaround for this? I mean to ignore a folder when one clones the repo?

**Comment by Saksham20** [comment]
or we just skip CI and have tests files only

**Pull request merged**